cosmo_tester/test_suites/cluster/broker_management.py

```diff
@@ -19,6 +20,14 @@ def get_broker_listing(broker, prefix='rabbit@'):
     return brokers
 
 
+def manager_list_brokers(manager):
+    return json.loads(
+        manager.run_command(
+            'cfy cluster brokers list --json 2>/dev/null | cat'
```

Comment:
why do we need cat, again?

Reply:
Comment added to code- it's because of escape characters being added by cfy which I think we dug into when this was originally created.